src/getters/index.ts

```diff
@@ -558,6 +559,7 @@ export async function getOperatorById(
             variables: {
                 operatorId,
             },
+            fetchPolicy: force ? 'network-only' : void 0,
```

Comment:
In the future I'm gonna be doing one of the following:
- default to `network-only`, or
- add `force` flag to each similar utility.

Reason? Cache at too many levels (too many = anything other than 0 or 1).